postgresql-11.5/src/bin/psql/startup.c

```diff
@@ -171,9 +171,23 @@ main(int argc, char *argv[])
 
     refresh_utf8format(&(pset.popt.topt));
 
+    pset.popt_bteq.topt.format = PRINT_ALIGNED;
+    pset.popt_bteq.topt.border = 1;
+    pset.popt_bteq.topt.pager = 1;
+    pset.popt_bteq.topt.pager_min_lines = 0;
+    pset.popt_bteq.topt.start_table = true;
+    pset.popt_bteq.topt.stop_table = true;
+    pset.popt_bteq.topt.default_footer = true;
```

Comment:
do we need all this variables?

Reply:
yes, because we need to use a pset printQueryOptBteq my_popt_bteq